articles/automation/how-to/private-link-security.md

```diff
@@ -127,15 +127,15 @@ If the service consumer has RBAC permissions on the Automation resource, they ca
 
 ## Set public network access flags
 
-You can configure an Automation account to deny all public configurations and allow only connections through private endpoints to further enhance the network security. If you want to restrict access to the Automation account from only within the VNet and not allow access from public internet, you can set `publicNetworkAccess` property to `$true`.
+You can configure an Automation account to deny all public configurations and allow only connections through private endpoints to further enhance the network security. If you want to restrict access to the Automation account from only within the VNet and not allow access from public internet, you can set `publicNetworkAccess` property to `False`.
```

Comment:
The property value should be set to `$false`. Please change this.

Reply:
Thanks for catching that. Just made the change.